go.mod

```diff
@@ -3,6 +3,7 @@ module k8s.io/cloud-provider-vsphere
 go 1.16
 
 require (
+	github.com/fsnotify/fsnotify v1.4.9 // indirect
```

Comment:
did you run go mod tidy? I don't see go.sum change

Reply:
yes, this is after `go mod tidy`. You can see there is fsnotify at go.sum
```
gopkg.in/fsnotify.v1 v1.4.7/go.mod h1:Tz8NjZHkW78fSQdbUxIjBTcgA1z1m8ZHf0WmKUhAMys=
```